source/simulator/stokes_matrix_free.cc

```diff
@@ -256,7 +256,7 @@ namespace aspect
       // iterations of our two-stage outer GMRES iteration)
       if (do_solve_Schur_complement)
         {
-          // first solve with the bottom left block, which we have built
+          // first solve with the bottom right block, which we have built
```

Comment:
hmm, that was probably just copied from here: https://github.com/geodynamics/aspect/blob/edcb0fa34bfdf4a5a49d58a90112837dfb95501e/source/simulator/solver.cc#L274, which also still has that. I guess that should be a different pull request.

Reply:
true, see #5069